projects/plugins/jetpack/_inc/client/recommendations/timer/index.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import React, { useState, useEffect, useCallback, useRef, useMemo } from 'react';
+
+const Timer = ( { className, expiryDate } ) => {
+	const date = useMemo( () => new Date( expiryDate ), [ expiryDate ] );
```

Comment:
Value's in UTC. The `Date` constructor takes care of applying the timezone detected by the browser.